README.md

```diff
@@ -22,6 +22,5 @@ video broadcast from space.
 - Make a virtual CAN bus
   - `$ sudo ip link add dev vcan0 type vcan`
   - `$ sudo ip link set vcan0 up`
-- Run `$ python3 -m oresat_dxwifi`
-  - Can mock the SkyTraq by adding the `-m` flag
+- Run `$ python -m oresat_dxwifi`
```

Comment:
Things got complicated because I had already committed my changes locally but hadn't synced with GitHub yet. I started the merging on GitHub, but ended up having to do it via the command line locally anyways. I pulled from the news master and merged into my local camera branch, resolved the issues and pushed it. I believe I got the option to do it via commit or something else. I didn't really see the difference since the merge issues were resolved and everything seems to be in order in both branches on GitHub.